db/schema.rb

```diff
@@ -10,11 +10,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_09_21_081127) do
+ActiveRecord::Schema.define(version: 2021_09_21_082544) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "chomp_sessions", force: :cascade do |t|
+    t.string "name"
+    t.datetime "date"
+    t.string "unique_identifier"
+    t.string "status"
```

Comment:
@TeeWhyKay actually should we just make the Status into Completed and let it be boolean True or False?

Reply:
I think should still stick to string, just in case, we want to account for:
E.g. 
"pending"
"completed"
"failed"